src/board_controller/brainalive/brainalive.cpp

```diff
@@ -6,7 +6,22 @@
 #include "timestamp.h"
 
 // common constants
-#define BRAINALIVE_PACKET_SIZE 224
+#define BRAINALIVE_PACKET_SIZE 220
```

Comment:
nitpicking but I think it could be slightly better to use constexpr instead defines because all these params are used only inside single method and can be easily defined there with constexpr